src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -116,14 +124,14 @@ public static Logger getLogger() {
 
   public static void gameOverClearCache() {
     // Are static, clear so that we don't keep the data around after a game is exited
-    ProBattleUtils.clearData();
+    concurrentCalc.setGameData(null);
```

Comment:
Please do not use 'null' values, causes NPE. Optional.empty(), or better yet, add a "clearGameData()" method so no null needs to be passed in.